mappings/net/minecraft/block/Material.mapping

```diff
@@ -3,54 +3,76 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15912 breakByHand Z
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
+		COMMENT Material for blocks that are stone or made from it, and generally prefer to be broken by a pickaxe.
 	FIELD field_15915 BUBBLE_COLUMN Lnet/minecraft/class_3614;
-	FIELD field_15916 SAND Lnet/minecraft/class_3614;
+	FIELD field_15916 AGGREGATE Lnet/minecraft/class_3614;
+		COMMENT A material or structure formed from a loosely compacted mass of fragments or particles.
 	FIELD field_15917 SPONGE Lnet/minecraft/class_3614;
 	FIELD field_15918 REDSTONE_LAMP Lnet/minecraft/class_3614;
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
+		COMMENT Material for the various portal blocks.
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
 	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
+		COMMENT Not in use, but has foliage color.
 	FIELD field_15922 LAVA Lnet/minecraft/class_3614;
 	FIELD field_15923 LEAVES Lnet/minecraft/class_3614;
-	FIELD field_15924 PART Lnet/minecraft/class_3614;
+	FIELD field_15924 PARTIAL_BLOCK Lnet/minecraft/class_3614;
```

Comment:
Not sure this one is descriptive. I'd consider a slab a "partial block"

Reply:
What's this material for?

Reply:
well, would be more common to assume slabs share material with their full block counterparts.

This material is for things like redstone components, rails, torches, ladders, etc.

Reply:
A material does not describe a block shape. It describes a subset of its properties. PARTIAL_BLOCK is not a good name because it describes its shape rather than its properties.

Reply:
That is not how any material is named, except in cases of clarifying it from another similar property.

Reply:
What about `MOUNTED`? `SUPPORTED`?

Reply:
`SUPPORTED` would be fine, if we're ok ignoring flower pots and the potted plants.

Reply:
That's a bug, so SUPPORTED seems good to me https://bugs.mojang.com/browse/MC-127036

Reply:
Not confirmed as a bug, but we'll go with it then.

Reply:
Skulls tho?

Reply:
i think skull needs a block to attach to as well, so `SUPPORTED` works very well